cmd/logger.go

```diff
@@ -137,6 +156,27 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
+func printJSON(fatal bool, source, cause, stack, msg string, data ...interface{}) {
+	level := "Error"
+	if fatal {
+		level = "Fatal"
+		defer os.Exit(1)
+	}
+	log := &logEntry{
+		Level:   level,
+		Source:  source,
+		Stack:   stack,
+		Cause:   cause,
+		Time:    time.Now().UTC().Format(time.RFC3339Nano),
+		Message: fmt.Sprintf(msg, data...),
+	}
+	buf := new(bytes.Buffer)
+	enc := json.NewEncoder(buf)
+	enc.SetEscapeHTML(false)
```

Comment:
this is not required IMO.  consumer knows it is JSON encoded.

Reply:
Without this, characters like `<` , `>` , `&` appear as their UTF encoded value

Reply:
yes. JSON is not for human.  What is the problem?

Reply:
For e.g, If the text within a JSON field is `<autogenerated>`, this will show up as `\u003cautogenerated\u003e` . If this is fine, I can do that.

Reply:
thats fine because consumer knows how to read e.g. `jq`

Reply:
done